### ISSUE TITLE
DB creates file of 25 GB

### ISSUE THREAD

**Issue by UnaFactura** [UnamSanctam/UnamWebPanel#223]
Hey. I am always running out of space on my hosting server because there is some file generated in db folder that reaches more than 20 GB in size. What is that?

**Comment by UnamSanctam** [comment]
Yes it's most likely due to checkpoint starvation, you can try the version 1.7.1 beta: [UnamWebPanel.zip](https://github.com/UnamSanctam/UnamWebPanel/files/10341755/UnamWebPanel.zip)
